### ISSUE TITLE
sync uses wrong path `\` characters when copying files to /pyboard while Python 3 is running on windows machines.

### ISSUE THREAD

**Issue by freemansoft** [BrianPugh/belay#78]
Was testing examples/06_external_modules_and_files with REPL across a Bluetooth Serial connection.  The sync operator appears to be doing using the backslash `\`  character in the path when copying from the windows filesystem to the device file system. 

Supposition:  `\` is the separator for the file system for Python on the PC picked up from the file/dir scan.

Test machine: Windows PC
Target device: Pico RP2040
Windows shell running "python 3" command: Powershell.  

Symptoms:
1. Test program copies files that end up prefixed with the backslash character, the windows root separator.
1. Program fails to find led.py 

Board directory after test fails.
```
C:\ESP8266-MicroPython> ls /pyboard/
\somemodule/       \led.py            \somemodule\led.py  \hello_world.txt
```

Expected results
```
C:\ESP8266-MicroPython> ls /pyboard/
somemodule/       led.py            somemodule/led.py   hello_world.txt
```

Board directory after manually restoring boot.py into the same device using ``rshell`` builtin cp ``cp boot-origi.py /pyboard`` command
```
C:\ESP8266-MicroPython> ls /pyboard/
\somemodule/       \led.py            \somemodule\led.py boot-orig.py       \hello_world.txt
```

**Comment by BrianPugh** [comment]
That's certainly an issue! I'll try and add Windows to the CI tests so I can catch and fix these. I should be able to get to it later this week.

**Comment by BrianPugh** [comment]
just updating to let you know that I'm working on this; the fix should be fairly simple, the harder part is just to get all the windows CI working so that I can confidently support windows/fix other issues that may come up. Should have a release done by end of weekend.

**Comment by freemansoft** [comment]
Thanks for the update
https://joe.blog.freemansoft.com/2023/01/an-open-source-developer-gave-best-in.html

**Comment by BrianPugh** [comment]
Should be fixed and addressed in belay `v0.16.1`. CI now additionally runs on a Windows environment and all unit/integration tests pass. My windows dev environment isn't the best, so if you notice other Belay issues, please open up another Issue!

**Comment by freemansoft** [comment]
Should I be able to verify this using the main branch?  `git pull`
I see the same behavior if I run the following on my windows machine
```
cd examples/06_external_modules_and_files
python3 main.py -p <COMPORT>
```

**Comment by BrianPugh** [comment]
it's working for me; are you sure you're using the absolute latest? What does `python3 -c "import belay; print(belay.__version__)"` output? If you previously `pip install belay`, it might be using an older version rather than the local git copy that you think it might be.

**Comment by freemansoft** [comment]
```
PS C:\Users\joe\Documents\GitHub\belay\examples\06_external_modules_and_file_sync> python3 -c "import belay; print(belay.__version__)"
0.16.0
```

pip uninstalled belay then reinstalled

```
Python 3.9.13 (tags/v3.9.13:6de2ca5, May 17 2022, 16:36:42) [MSC v.1929 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import belay
>>> print(belay.__version__)
0.16.1
>>>
```

**Comment by BrianPugh** [comment]
Is it working now that you're on 0.16.1?

**Comment by freemansoft** [comment]
Yes.  Sorry about the false alarm.

Still seeing that I have to do a reset to get the repl back after running 

```
PS C:\Users\...\Documents\GitHub\belay\examples\06_external_modules_and_file_sync> python3 main.py -p COM4     
Using synced "led.py" via explicit commands.
Using synced "led.py" via task decorator.
Using synced "somemodule/led.py" via task decorator.
Reading a synced file via task decorator. Contents:
Hello World
The Quick Brown Fox Jumped Over The Lazy Dog.

PS C:\Users\...\Documents\GitHub\belay\examples\06_external_modules_and_file_sync> rshell -p COM4
Using buffer-size of 32
Connecting to COM4 (buffer-size 32)...
Trying to connect to REPL .
```
hangs there

**Comment by freemansoft** [comment]
Still seeing that I have to do a reset to get the REPL back after running example 6

```
PS C:\Users\...\Documents\GitHub\belay\examples\06_external_modules_and_file_sync> python3 main.py -p COM4     
Using synced "led.py" via explicit commands.
Using synced "led.py" via task decorator.
Using synced "somemodule/led.py" via task decorator.
Reading a synced file via task decorator. Contents:
Hello World
The Quick Brown Fox Jumped Over The Lazy Dog.

PS C:\Users\...\Documents\GitHub\belay\examples\06_external_modules_and_file_sync> rshell -p COM4
Using buffer-size of 32
Connecting to COM4 (buffer-size 32)...
Trying to connect to REPL .
```
hangs there

**Comment by freemansoft** [comment]
This issue is closed. I can open another one about the REPL loss in Example06

**Comment by BrianPugh** [comment]
I've got a fix incoming for that as well.

**Comment by BrianPugh** [comment]
Fixed now in Belay v0.16.2
